### PR TITLE
ready with task1 from IO

### PR DESCRIPTION
**Pull request by ivanP91**
creat pullrecuest

### CODE REVIEW

**Comment by mgenov** [comment]
Pull Request

File: src/com/clouway/task3/ConsoleWriteInTextFail.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.clouway.task3;
+
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class ConsoleWriteInTextFail {
```
Comment (mgenov) [inline src/com/clouway/task3/ConsoleWriteInTextFail.java]: What is this ?

File: src/com/clouway/task1/ReadeConsole.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.clouway.task1;
+
+/**
+ * Created by clouway on 09.05.16.
+ */
+
+import java.util.Scanner;
+
+public class ReadeConsole {
```
Comment (mgenov) [inline src/com/clouway/task1/ReadeConsole.java]: Think for a better name of this class.

File: src/com/clouway/task3/ConsoleWriteInTextFail.java (hunk 1 above)
Comment (ivanP91) [inline src/com/clouway/task3/ConsoleWriteInTextFail.java]: I wrong at adding ,this is from next task.

File: src/com/clouway/task1/ReadeConsole.java (hunk 2 above)
Comment (ivanP91) [inline src/com/clouway/task1/ReadeConsole.java]: Okay, i will think.

File: src/com/clouway/task1/Writer.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.clouway.task1;
+
+/**
+ * Created by clouway on 09.05.16.
+ */
+
+import java.util.Scanner;
+
+public class Writer {
```
Comment (mgenov) [inline src/com/clouway/task1/Writer.java]: You are doing something wrong here. What is it ?

File: src/com/clouway/task1/Demo.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.clouway.task1;
+
+import java.io.IOException;
+
+/**
+ * Created by clouway on 09.05.16.
+ */
+public class Demo {
+  public static void main(String[] args) throws IOException {
+    Writer readeConsole = new Writer();
```
Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: Why Writer is reading from console?

Comment (ivanP91) [inline src/com/clouway/task1/Demo.java]: i fixed name of ReadFromConsole

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: Are you sure that this would be a correct name for a class? Doesn't sound good to me.

Comment (ivanP91) [inline src/com/clouway/task1/Demo.java]: InputConsole 
how it sounds?

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: Try something different.

Comment (ivanP91) [inline src/com/clouway/task1/Demo.java]: ConsoleReader 
is it okay ?

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: No, I want something else, cause everyone is currently using this one and I prefer to be different.

Comment (ivanP91) [inline src/com/clouway/task1/Demo.java]: What you think for inputStream ?

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: Java already have such class.

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: Also it's binary stream, but you read strings and ints, so seems totally wrong.

Comment (ivanP91) [inline src/com/clouway/task1/Demo.java]: ScannerInput 
what you think?

Comment (mgenov) [inline src/com/clouway/task1/Demo.java]: ok, make it ScannerInput.

**Pull request closed without merging**